run/qe/x0.scf.in

```diff
@@ -42,9 +42,7 @@ ATOMIC_SPECIES
  C  12.01  C_pbe_v1.2.uspp.F.UPF
  H  1.008  H.pbe-rrkjus_psl.0.1.UPF
 
-
-K_POINTS automatic
-1 1 1  0 0 0                  
+K_POINTS gamma
```

Comment:
no, we should not use Gamma in this format!  This prints only a half of the wavefunction and it would need be reconstructed

Reply:
There is a wavefunction restoration process for gamma point calculation in the read_qe_wfc function of the QE_methods.py module.

Reply:
Ah, ok. By the way, could you check that these two options produce consistent results (just few steps of MD - compare NACs)?

Reply:
Okay, I will check that out.